resources/Microsoft.VSCode.Dsc/Microsoft.VSCode.Dsc.psm1

```diff
@@ -106,115 +114,124 @@ class VSCodeExtension
     [DscProperty()]
     [bool] $Exist = $true
 
+    [System.Management.Automation.HiddenAttribute()]
+    [DscProperty()]
+    [System.Boolean]
+    $UseInsiders = $false
+
+    [System.Management.Automation.HiddenAttribute()]
+    [DscProperty()]
+    [System.String]
+    $CustomPath
+
     static [hashtable] $InstalledExtensions
 
-    static VSCodeExtension()
-    {
-        [VSCodeExtension]::GetInstalledExtensions()
+    VSCodeExtension() {
     }
 
-    static [VSCodeExtension[]] Export()
+    VSCodeExtension([string]$Name, [string]$Version) {
+        $this.Name = $Name
+        $this.Version = $Version
+    }
+
+    # TODO: validate if 'dsc.exe' is able to parse in the 'VSCodeExtension' class with UseInsiders input.
+    static [VSCodeExtension[]] Export([System.Boolean]$UseInsiders, [System.String]$CustomPath)
```

Comment:
This method no longer needs to be static anymore since you're now calling it inside the get method.
```suggestion
    [VSCodeExtension[]] Export([bool]$UseInsiders)
```